content/05_games/05_sourcemods.md

```diff
@@ -0,0 +1,16 @@
+### Sourcemods
+
+#### Timing
+
+- Timing begins when you have gained control of the player.
+- The ending timing will be listed with the category.
+- Exceptions will be listed if there are any.
+
+#### General Rules
+
+`Developer 1` cannot be used to make the player move early, any runs that used
+developer 1 will be grandfathered in, and 1.5 seconds will be added per map where it was deemed to be an advantage.

```

Comment:
The rules page lists the rules *in their current form*, and is not concerned with the details of historical rule changes. Also, don't capitalize commands.
"`developer 1` cannot be used to make the player move early."